src/app/Providers/ProvidersPage.tsx

```diff
@@ -56,14 +69,9 @@ const ProvidersPage: React.FunctionComponent = () => {
     new Set(clusterProviders.map((provider) => provider.spec.type))
   ).filter((type) => !!type) as ProviderType[];
 
-  const [activeProviderType, setActiveProviderType] = React.useState<ProviderType | null>(
-    availableProviderTypes[0]
-  );
-  React.useEffect(() => {
```

Comment:
We still need the useEffect here because when the component first mounts at `/providers` the `match` won't be defined, so we end up with no tab selected.